torch/ao/quantization/_pt2e/quantizer/qnnpack_quantizer.py

```diff
@@ -255,13 +258,15 @@ def annotate(self, model: torch.fx.GraphModule) -> torch.fx.GraphModule:
             # supported op type.
             self._annotate_conv2d_relu(node, global_config)
             self._annotate_conv2d(node, global_config)
-            self._annotate_linear(node, global_config)
+            self._annotate_linear3d(node, global_config)
+            self._annotate_linear2d(node, global_config)
```

Comment:
This is exactly what I dont like, but anywho. We need to close on module based stuff soon

Anyways we should put all of that innside annotate_linear.

Reply:
yeah sure